### ISSUE TITLE
Visualiser with FP64 `x`/`y`/`z` runtime error

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU2#1217]
Using doubple precision (64 bit) floating point values for X/Y/Z (and presumably glm position) types results in an error at runtime in `ModelVis.cpp`. 

```
Agent 'agent' has not had x, y or z variables set, agent requires location to render, in ModelVis::activate()
```

https://github.com/FLAMEGPU/FLAMEGPU2/blob/4e0260476353710860a4c29c7bd96da27c80aa05/src/flamegpu/visualiser/ModelVis.cpp#L134

This might actually need a change in the vis repo, where `core_text_buffers` is set, but the error is thrown from ModelVis.cpp in this repo.

**Comment by Robadob** [comment]
Yeah, will need implementing on the visualiser side. May need to do some type conversion, not too sure about double precision floats in shader texture buffers.
